py/torch_tensorrt/dynamo/conversion/ops_evaluators.py

```diff
@@ -35,3 +36,14 @@ def generic_evaluator(
         f"Evaluating {ConverterRegistry.qualified_name_or_str(target)} on object with name: {name}"
     )
     return target(*args)
+
+
+@dynamo_tensorrt_converter(torch.ops.aten.arange.start_step)
+def aten_ops_arange_start_step(
+    ctx: ConversionContext,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return np.arange(*args)
```

Comment:
A very minor comment - using `np.arange(*args)` requires the schemas of `arange.start_step` and `np.arange` to be the same position-wise. In this case they are the same and are unlikely to change, so I think this is fine, just wanted to mention this scheme may not hold between Torch and Numpy in general.